dask-metrics/dask_metrics/visualize.py

```diff
@@ -0,0 +1,102 @@
+import cudf
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Can make this an optional import and handle the case where matplotlib is not installed in the environment.